README.md

```diff
@@ -54,38 +57,71 @@ Clone this repository to your desired folder.
 
 ## Prerequisites
 
-In order to run this project, you need:
+In order to run this project, you need to have Node.js and npm (Node Package Manager) installed on your machine.
 
 ## Install
 
-Install this project with:
+To install this project and its dependencies, follow these steps:
+
+[```bash
+npm install]
+
 
-## Usage
+### Usage
 
-To run the project, execute the following command:
+To use this project, create a new branch to add more features. However, to run the project, use the following commands:
+
+npm run build   # It will create a production-ready file
+npm start       # It will run the server on your machine
 
 ## Run tests
 
-To run tests, use the following command:
+This project includes linters or tests to avoid any kind of typos and ensure good practices. To run the linter tests on the HTML or any HTML file, use the following command:
+
+npx hint .
+
+To run the linter tests on stylesheets, use the following command:
+
+npx stylelint "*+/*.{css, scss)"
+
+To run the linter tests on JavaScript files, use the following command:
+
+npx eslint .
 
 ## Deployment
 
-You can deploy this project using:
+You can deploy this project using the deployment method of your choice.
 
 # 👥 Authors <a name="authors"></a>
 
-👤 **CVILLA09**
+👤 Cristian Villa
 
 - GitHub: [@CVILLA09](https://github.com/CVILLA09)
 
 # 🔭 Future Features <a name="future-features"></a>
 
 There are no planned future features for this project.
 
```

Comment:
- [ ] As mentioned by the previous reviewer, It would be beneficial to showcase your proactive approach by including 1-3 future features in your project. 
- [ ] This demonstrates your forward-thinking mindset and dedication to continuously improving your work.🥱🥱